chat/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils import timezone
+
+
+# class ChatManager(models.Manager):
+#     def get_group_chat(self, group_id):
+#         return self.filter(group_id=group_id).order_by('date_time')
+#
+#     def get_student_course_chat(self, student_course_id):
+#         return self.filter(student_course_id=student_course_id).order_by('date_time')
+
+
+class Message(models.Model):
+    message_id = models.BigAutoField(primary_key=True)
```

Comment:
I think `AutoField` is sufficient for the ID instead of `BigAutoField`

Reply:
I wanted to be consistent with the other PRs.
In addition, in the case of chat where there are lots of instant messages, it is possible to surpass the autofield limit (in theory) compared to other models. 
What do you think?

Reply:
Sounds reasonable! No need to change then.